package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/graph-jira",
-  "version": "3.8.4",
+  "version": "3.9.0",
```

This repo is versioned using Auto. Info can be found in the README but in short, do not change this number manually and tag this PR with `minor` and `release`
```suggestion
  "version": "3.8.4",
```

---

https://github.com/JupiterOne/graph-jira#versioning-this-project